setup.py

```diff
@@ -136,43 +136,47 @@ def read(*names, **kwargs):
     )
 
     ext_modules = [
-        CUDAExtension('kaolin.cuda.load_textures', [
-            'kaolin/cuda/load_textures_cuda.cpp',
-            'kaolin/cuda/load_textures_cuda_kernel.cu',
-        ]),
-        CUDAExtension('kaolin.cuda.sided_distance', [
-            'kaolin/cuda/sided_distance.cpp',
-            'kaolin/cuda/sided_distance_cuda.cu',
-        ]),
-        CUDAExtension('kaolin.cuda.furthest_point_sampling', [
-            'kaolin/cuda/furthest_point_sampling.cpp',
-            'kaolin/cuda/furthest_point_sampling_cuda.cu',
-        ]),
-        CUDAExtension('kaolin.cuda.ball_query', [
-            'kaolin/cuda/ball_query.cpp',
-            'kaolin/cuda/ball_query_cuda.cu',
-        ]),
-        CUDAExtension('kaolin.cuda.three_nn', [
-            'kaolin/cuda/three_nn.cpp',
-            'kaolin/cuda/three_nn_cuda.cu',
-        ]),
-        CUDAExtension('kaolin.cuda.tri_distance', [
-            'kaolin/cuda/triangle_distance.cpp',
-            'kaolin/cuda/triangle_distance_cuda.cu',
-        ]),
-        CUDAExtension('kaolin.cuda.mesh_intersection', [
-            'kaolin/cuda/mesh_intersection.cpp',
-            'kaolin/cuda/mesh_intersection_cuda.cu',
-        ]),
-        CUDAExtension('kaolin.graphics.nmr.cuda.rasterize_cuda', [
-            'kaolin/graphics/nmr/cuda/rasterize_cuda.cpp',
-            'kaolin/graphics/nmr/cuda/rasterize_cuda_kernel.cu',
-        ]),
         triangle_hash_module,
         mise_module,
         mcubes_module,
         nnsearch_module,
     ]
+    
+    # If building with readthedocs, don't compile CUDA extensions
+    if os.getenv('READTHEDOCS') != 'True':
```

Comment:
Is this the recommended way to check if we're building on readthedocs?

Reply:
Not sure if there is a recommended way but happy to change it if you have a alternative suggestion. This works well and can be easily extended to include a CUDA check once we support CPU-only installation.

Reply:
Ah, it appears this is the recommended way afterall: https://docs.readthedocs.io/en/stable/faq.html#how-do-i-change-behavior-for-read-the-docs